al_run_list/get_run_list.py

```diff
@@ -21,11 +21,10 @@ def connectMySQL(run_num):
     '''Read in yaml configuration file and return a connection using the data from the file.'''
 
     #Open connection and read it into directory
-    mysql_db = None
```

Comment:
It looks like yaml.safe_load assigns None if it fails? If so no need to 'instantiate' this.